docs/design/developer_notes/Transitions.notes.md

```diff
@@ -41,4 +41,138 @@ we are going to have a configurable 'delete' for the transitions; what does this
  * What do we do about reservations? how long are transitions valid for?  b/c the state kinda goes stale... e.g.  the transition has completed; but the thing may have been rebooted now. I believe our conclusion was that we were going to all for deputy keys to be passed in to us, and the reason why we call it 'transitions' and not 'state'. we will 'stop looking' after we get one positive confirmation; Its impossible to guarantee the hardware state; because its not always 100% reliable. Too much jitter in the hardware.
  * we arent putting a hierarchy flag in the API; but rather hierarchies can be built in the CLI.  the API accepts xnames.  They ONLY hierarchy stuff PCS should intervene with is when hardware damage is possible. So we need to be power relationship aware: 'whats plugged into what'; but we wont by default assume a set of transitions 'inherently' - people have to be explicit. 
  * an important use case to remember is it take a `force-off` to chassisBMCs to clear an EPO.
- * Administrators do not want management stuff to go down unless they say so intentionally! This is where locking/reservation API is useful... Not sure what role the CLI plays in this... maybe a 'are you sure' type of thing?
\ No newline at end of file
+ * Administrators do not want management stuff to go down unless they say so intentionally! This is where locking/reservation API is useful... Not sure what role the CLI plays in this... maybe a 'are you sure' type of thing?
```

Comment:
FYI: [CASMHMS-3234](https://jira-pro.its.hpecorp.net:8443/browse/CASMHMS-3234)

Reply:
It didn't change or create this line. This is just part of the overall notes on transitions predating CASMHMS-3234